Description: Fix t/04_reconnect.t
 The reconnect test fails due to a race condition. This patch sleeps for
 7 seconds (randomly chosen), and the test passes.
Author: Marius Gavrilescu <marius@ieval.ro>
Forwarded: no
Last-Update: 2013-12-25
--- a/t/04_reconnect.t
+++ b/t/04_reconnect.t
@@ -1,6 +1,7 @@
 use Test::More;
 use Test::TCP;
 use AnyEvent::Gearman::Client;
+use Time::HiRes qw/sleep/;
 
 eval q{
         use Gearman::Worker;
@@ -43,6 +44,7 @@
         ok !$js->connected, 'disconnect ok';
 
         $js->dead_time(0); # trick for reconnect soon
+        sleep 0.3;
     }
 
     {
